roles/firmware/templates/iiab-firmware-warn.sh

```diff
@@ -2,8 +2,9 @@
 
 if [ -f /tmp/.fw_modified ]; then
     echo -e "\n\e[41;1mWiFi Firmware link(s) modified, per iiab/iiab#2853: PLEASE REBOOT!\e[0m"
-    echo
-    echo -e "If you want this warning to stop, run: sudo rm /tmp/.fw_modified\n"
+    # /tmp should be auto cleaned with a reboot
```

Comment:
Both options / explanations should be visible:

```
If you want this warning to stop, run 'sudo rm /tmp/.fw_modified' or reboot.
```

Operators get very unhappy when loud warnings appear without explanation, or worse explanation(s) hidden from them.

(And even more so if firmware-install-on-each-boot glitch were to occur at anytime in future.)

Reply:
No the correct fix is to reboot, period the firmware was replaced. As you know /tmp/.fw_modified doesn't survive a reboot. removing that file just suppress the warning from other new logins so what is gained when the only correct answer is to reboot, this is not a suggestion.    This warning should really be extended to the desktops and not just ssh sessions.

Reply:
The explanation should explain clearly, one way or another.  In that 2nd line (it's more white not red) that serves an important purpose (UX transparency).  Example:

```
If you want this warning to stop, reboot to remove /tmp/.fw_modified
```

Reply:
Look If you wanted the second line to say 'sudo rm /tmp/.fw_modified' or reboot' or whatever you can change it in the tree now and I'll rebase, just keep the changes small given the largeness of this PR that I would like to get in quickly.

Reply:
Done:

https://github.com/iiab/iiab/commit/a2cbfc45d6f558e493272d4d8fb93c91b2a4275e